functions-iterators/starter.js

```diff
@@ -29,27 +29,58 @@ This will help us track your progress and it will give you practice writing good
 
 // PART 0: Just DO it!!
 
-
+function justDoIt(name){
+  console.log(`${name}, JUST DO IT !!`);
+}
 
 // PART 1: Big or Small String?
 
-
+function bigOrSmallString (myName){
+  if (myName.length > 10){
+    console.log (" This Worf id loooong 😬")
+  }else{ console.log (" This Word is Small");
+  }
+  
+} console.log(bigOrSmallString('myName'));
 
 // PART 2: Odd or Even String Length?
 
+function oddOrEvenString (myString){
+  if (myString.length %2==0){
+    console.log ("This String is TOTALLY Even");
+}else { if (myString.length%2 != 0)
+console.log (" This is Odd ");
+}
+}
 
 
 // PART 3: Median
 
+function medianOfArray (numArray){
 
+sort(numArray);
```

Comment:
sort is an array method to use it writhe the array name first like this `numArray.sort()`, there is a lot of logic error but I get what you trying to achieve your code should look like this 
```
 numArray.sort(function (a, b) { return a - b }); 
 if (numArray.length % 2 == 0){
      console.log(
     ( numArray[Math.floor(numArray.length/2)] + 
       numArray[Math.floor(numArray.length/2) - 1] ) /2 );
}else{
    console.log( numArray[ Math.floor(numArray.length/2)] );
}

```

we need to used `Math.floor` because array index only takes positive integer numbers